src/main/sensors/rangefinder.c

```diff
@@ -73,6 +73,7 @@ PG_REGISTER_WITH_RESET_TEMPLATE(rangefinderConfig_t, rangefinderConfig, PG_RANGE
 
 PG_RESET_TEMPLATE(rangefinderConfig_t, rangefinderConfig,
     .rangefinder_hardware = RANGEFINDER_NONE,
+    .rangefinder_trust = 0,
```

Comment:
Need to bump PG version on line 72 to 1

![image](https://github.com/user-attachments/assets/9bcd49c9-569d-46b8-a53a-1eee5e7b0364)

Reply:
(It's only BF policy. PG can handle this case correctly)

Reply:
Sorry, writing documentation is not my strong point..

Simple PG are first set to zero, then initialized to defaults and then overwritten with saved version. https://github.com/betaflight/betaflight/blob/master/src/main/pg/pg.c#L78. Struct members added to end of pg struct are thus initialized to default and version increase is necessary only when changing PG layout or interpretation of values.

But this mechanism was never finished for array and possibly profile entries. It does support only change in array length (number of items). Necessary plumbing is almost there ..

This caused problem with array PG changes (sometimes poorly diagnosed, sometimes causing quite nasty bugs) and it was decided that is is safer to always change PG version on change.